compiler/rustc/build.rs

```diff
@@ -0,0 +1,31 @@
+use std::path::PathBuf;
+use std::{env, fs};
+
+fn main() {
+    let target_os = env::var("CARGO_CFG_TARGET_OS");
+    let target_env = env::var("CARGO_CFG_TARGET_ENV");
+    if Ok("windows") == target_os.as_deref() && Ok("msvc") == target_env.as_deref() {
+        set_windows_exe_options();
+    }
+}
+
+// Add a manifest file to rustc.exe and increase the main thread's stack size.
+fn set_windows_exe_options() {
+    const STACK_SIZE: u32 = 2 * 1024 * 1024;
+    static WINDOWS_MANIFEST_FILE: &str = "Windows Manifest.xml";
+
+    let mut manifest = PathBuf::from(env::var_os("OUT_DIR").unwrap());
+    manifest.push(WINDOWS_MANIFEST_FILE);
+    fs::copy(WINDOWS_MANIFEST_FILE, &manifest).unwrap();
```

Comment:
Why is it copied?

Reply:
Oh yeah, that's completely unnecessary. I'll fix.